server/middleware/auth.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-else-return */
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+import jwt from 'jsonwebtoken';
+import config from '../config';
+
+function loginResponse(response) {
+  return response.status(401).json({
+    success: false,
+    message: 'Please login first',
+  });
+}
+
+function unauthorizedResponse(response) {
```

Comment:
merge this function with above function and pass message parameter to the function